content/07.pathogenesis.md

```diff
@@ -26,14 +34,17 @@ Promisingly though, sera from convalescent SARS patients inhibited SARS-CoV-2 vi
 
 #### Viral Replication and Spreading
 
-- Basic introduction into replication cycle
+<!-- - Basic introduction into replication cycle
```

Comment:
Within cells and between cells feels like virology to me. Between people feels like transmissibility. It would probably be good to focus pathogenesis on "how does this lead to bad things happening?" That is in this section, but it seems somewhat overshadowed by these other topics.

I guess I'm saying, I am not a fan of introducing the things that are in these comments right here. The text itself seems solid.

Reply:
I think this is addressed [in changes here](https://github.com/greenelab/covid19-review/pull/620/commits/9f20d4d135f6d7083d83976158ef15644a7cdd84) (Multiple comments handled in single commit)